lib/dialects/sqlite/connection-manager.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+const path = require('path');
+const jetpack = require('fs-jetpack');
```

Comment:
Do we really need to use this new library, its unpacked size is `~300KB`

Reply:
@sushantdhiman You're right. This package greatly simplifies the work with the file system, makes the code more concise. In addition, `sequelize` included it as `devDependency`. But I did not pay attention to its size. This in my opinion is not critical on the server side, but due to the fact that the file system is used by the `sequelize` only in ~ one place, I would like to do the following:

1. Return the `fs-jetpack` package back to `devDependencies` and use it only for the convenience of writing tests
2. Write my own function(similar to `jetpack.dir()`) to automatically provide the path for `sqlite.options.storage`

Reply:
It is only used for creating a path `jetpack.dir(path.dirname(options.storage))`, right? With Node 10 `mkdirSync` supports this https://nodejs.org/docs/latest-v10.x/api/fs.html#fs_fs_mkdirsync_path_options

Reply:
Awesome! Since `sequelize` **6.x** will support node version **10** and more, we can actually use `fs.mkdir(path, { recursive: true })`. Can I do a pull request?

Reply:
That will be great @multum

Reply:
Oops, nice catch @sushantdhiman, sorry about that.